src/Tizen.NUI/src/internal/Transition/TransitionItemBase.cs

```diff
@@ -78,7 +80,7 @@ public AlphaFunction AlphaFunction
         /// <summary>
         /// Gets or sets whether the View moves with child or not.
         /// </summary>
-        public bool TransitionWithChild
+        public virtual bool TransitionWithChild
```

Comment:
can I ask why `virtual` is added? not like previous patch?

Reply:
Because in this patch, TransitionGroupItem override the property.